apps/client/sttr_build/fetch.ts

```diff
@@ -0,0 +1,89 @@
+import { emptyDir, exists, writeJson } from "https://deno.land/std/fs/mod.ts";
+import { join } from "https://deno.land/std/path/mod.ts";
+import { assert } from "https://deno.land/std/testing/asserts.ts";
+import { makeRunWithLimit } from "https://denopkg.com/alextes/run-with-limit/mod.ts";
+
+import { APIConfig, ApiResult } from "./types.d.ts";
+
+const maxAPIConnections = 6; // XXX convert to cli argument
+
+// XXX npm run sttr moet deno aanroepen
+// XXX document...
+if (Deno.args.length < 2) {
+  console.error(`
+Dev usage:
+$ deno run --unstable --allow-net --allow-write --allow-read fetch.ts BASE_DIR API_KEY --config=config.local.ts (default=config.ts)
+`);
+  Deno.exit();
+}
+
+const baseDir = join(Deno.cwd(), Deno.args[0]);
+const headers = {
+  "x-api-key": escape(Deno.args[1]),
+};
+// baseDir should exist
+assert(await exists(baseDir));
+
+const { apis }: { apis: APIConfig[] } = await import(
+  join(Deno.cwd(), "config.local.ts")
+); // XXX convert to cli argument
```

Comment:
Lots of Amsterdam kisses (xxx) in this file

Reply:
Still needs some work I guess 😜